src/main/java/io/ebean/config/dbplatform/DbDefaultValue.java

```diff
@@ -63,8 +75,128 @@ public String convert(String dbDefaultLiteral) {
     if (dbDefaultLiteral == null) {
       return null;
     }
+    if (dbDefaultLiteral.startsWith("$RAW:")) {
+      return dbDefaultLiteral.substring(5);
+    }
     String val = map.get(dbDefaultLiteral);
     return val != null ? val : dbDefaultLiteral;
   }
 
+  
+  /**
+   * This method checks & convert the {@link DbDefault#value()} to a valid SQL literal.
+   * 
+   * This is mainly to quote string literals and verify integer/dates for correctness.
+   * <p>
+   * Note: There are some special cases:
+   * </p>
+   * <ul>
+   *    <li>Normal Quoting: <code>@DbDefault("User's default")</code> on a String propery
+   *        returns: <code>default 'User''s default'</code><br/>
+   *        (the same on an integer property will throw a NumberFormatException)</li>
+   *    <li>Special case null: <code>@DbDefault("null")</code> will return this: <code>default null</code><br/>
+   *        If you need really the String "null", you have to specify <code>@DbDefault("'null'")</code>
+   *        which gives you the <code>default 'null'</code> statement.</li>
+   *    <li>Any statement, that begins and ends with single quote will not be checked or get quoted again.</li>
+   *    <li>A statement that begins with "$RAW:", e.g <code>@DbDefault("$RAW:N'SANDNES'")</code> will lead to 
+   *        a <code>default N'SANDNES'</code> in DDL. Note that this is platform specific!</li>
+   * </ul>
+   */
+  public static String toSqlLiteral(String defaultValue, Class<?> propertyType, int sqlType) {
```

Comment:
This is a bit tricky to explain. I have to convert the @DbDefaut(value="foo") to a valid string / boolean / number literal.